src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;

```

Comment:
When creating a new instance of RightTriangle, consider using separate variables for 'firstLeg' and 'secondLeg' before passing them to the constructor for better code readability.